cognite/powerops/client/shop/shop_run_api.py

```diff
@@ -31,7 +33,40 @@ def __init__(self, client: CogniteClient, dataset_id: int, cogshop_version: str
         self._dataset_id = dataset_id
         self.cogshop_version = cogshop_version
 
-    def trigger_case(self, case: SHOPCase, source: str | None = None) -> SHOPRun:
+    def trigger_case(self, case: Case, concurrent_calls: int = 5) -> list[SHOPRun]:
+        """
+        Trigger a collection of shop runs related to one Case (also referred to as watercourse).
+        For each ShopCase the prerun file will be used to trigger a shop run event in cdf,
+        and used to trigger the CogShop container.
+        Prerun files must exist in CDF.
+        """
+        shop_events = []
+        for shop_run in case.pre_runs:
+            now = datetime.datetime.now(datetime.timezone.utc)
```

Comment:
Lets have the same time for all shop events, and limit ourselves to seconds. 
```suggestion
        shop_events = []
        now = datetime.datetime.now(datetime.timezone.utc).replace(microseconds=0)
        for shop_run in case.pre_runs:
```